packages/block-library/src/social-link/block.json

```diff
@@ -9,13 +9,15 @@
 	"textdomain": "default",
 	"attributes": {
 		"url": {
-			"type": "string"
+			"type": "string",
+			"role": "content"
 		},
 		"service": {
 			"type": "string"
 		},
 		"label": {
-			"type": "string"
+			"type": "string",
+			"role": "content"
 		},
 		"rel": {
 			"type": "string"
```

Comment:
So the button block has `rel` as `role: content`, but the image block doesn't, which is a bit of a quandry. I think it maybe makes sense for it to have a `content` role, as it's part of the url which is also content. If the user can edit a url, they may want to also set `rel` and `openInNewTab`, so those have to be `contentOnly` too.

On the social link block, the rel can't be edited in contentOnly mode (it's in the inspector > advanced area), so perhaps it should be a follow-up issue to make that work in addition to adding the role.